frontend/src/components/Calendar/Points.module.css

```diff
@@ -0,0 +1,37 @@
+.taskList {
+    list-style-type: none;
+    padding: 0;
+}
+  
+.tasks {
```

Comment:
- Maybe we can add a little more margin here? The tasks block seem to be a bit crunch together

Reply:
Done!